strings/reverse_letters.py

```diff
@@ -1,19 +1,22 @@
-def reverse_letters(input_str: str) -> str:
+def reverse_letters(sentence: str, length: int) -> str:
```

Comment:
```suggestion
def reverse_letters(sentence: str, length: int = 0) -> str:
```
Let's make the length an optional parameter so that we can still preserve the same functionality as the original `reverse_letters`

Please also add a test case for `length = 0`